src/test/java/nl/fuchsia/model/PersoonTest.java

```diff
@@ -0,0 +1,90 @@
+package nl.fuchsia.model;
+
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import javax.validation.ConstraintViolation;
+import javax.validation.Validation;
+import javax.validation.Validator;
+import javax.validation.ValidatorFactory;
+import java.time.LocalDate;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.Assert.assertEquals;
+
+public class PersoonTest {
+
+    private static Validator validator;
+    private static Persoon persoon;
+
+    @BeforeClass
+    public static void setupAll() {
+        ValidatorFactory factory = Validation.buildDefaultValidatorFactory();
+        validator = factory.getValidator();
+    }
+
+    @Before
+    public void setup() {
+        persoon = new Persoon(2, "Rense", "Houwing", "Voltawerk2", "36", "8401 EN", "Gorredijk", "123456789", LocalDate.of(1967, 10, 12));
+    }
+
+    /**
```

Comment:
Moet dit in Javadoc omschreven worden?

Reply:
De naam van de methode zegt ook reeds voldoende. Javadoc's aangeapst.